Delicate-Butterflies/Bangazon-Site
# Sb double checking boilerplate

Related Issue: #28

Description: Updated user model to reflect ours. Also changed config.json to no longer be tracked by git.

Chai/Mocha Tests File: -

PR Testing Steps:
- pull it down
- changed you config database to whatever you want
- check to ensure your config is no longer being tracked with a ```git status``` (it shouldn't show up)
- visually compare user model to ERD 

Check list before you make a PR:
